src/scss/settings/_colors.scss

```diff
@@ -31,13 +29,12 @@
   *
   * An some gradients defined
   *
-  * @iframe ../../demo/variables/colors/gradients/demo.html
-  *
+  * @iframe ../../demo/variables/colors/gradients/demo.html 260
```

Comment:
@GeorgeJegon here, the path could be `demo/gaiden-css/variables/colors/gradients/demo.html`